controllers/admin/index.php

```diff
@@ -0,0 +1,32 @@
+<?php
```

Comment:
file to remove. Added by package script.

Reply:
removed